src/esgvoc/api/data_descriptors/frequency.py

```diff
@@ -1,7 +1,39 @@
+"""
+Model (i.e. schema/definition) of the frequency data descriptor
+"""
+
 from esgvoc.api.data_descriptors.data_descriptor import PlainTermDataDescriptor
 
 
 class Frequency(PlainTermDataDescriptor):
```

Comment:
Karl proposed calling this ReportingInterval, which is technically more accurate. I'm not sure the change is worth making though.